COCO_DATA_FORMAT.md

```diff
@@ -112,7 +112,7 @@ Here is one example of the json file for the image regression task, where the "t
 
 ## Image retrieval
```

Comment:
This basically make it almost the same with image-text matching task + image classification

Reply:
https://github.com/microsoft/vision-datasets/blob/main/COCO_DATA_FORMAT.md#image-text-matching

Reply:
Agree. I'll abandon this PR. In addition, I'll prepare a proposal for a new Image Retrieval dataset type to replace this one and schedule a design review to discuss and align